landcover.mss

```diff
@@ -474,7 +493,7 @@
     [way_pixels >= 4]  { polygon-gamma: 0.75; }
     [way_pixels >= 64] { polygon-gamma: 0.3;  }
   }
- 
+
```

Comment:
whitespace noise

Reply:
This has already been removed as part of a whitespace cleanup